src/odemis/gui/cont/acquisition.py

```diff
@@ -1500,6 +1505,13 @@ def on_acquisition(self, evt):
         self._show_status_icons(None)
         self._tab_panel.Layout()  # to put the gauge at the right place
 
+        # fold the streams if possible
+        folds = tuple(acqmng.foldStreams(self._tab_data_model.acquisitionStreams))
+
+        if len(self._tab_data_model.acquisitionStreams) > len(folds):
+            self.lbl_acqestimate.SetLabel("EBIC and CL streams acquired simultaneously")
+            self._tab_data_model.acquisitionStreams = folds
```

Comment:
Isn't this going to cause issues afterwards when deleting the streams, or acquiring new streams? 
I think it's simpler to not change `.acquisitionStreams` pass `folds` to `acqmng.acquire()`